app/assets/stylesheets/admin.scss

```diff
@@ -32,9 +31,34 @@
   }
 }
 
+// Weekly stats
+.week-selector {
+  margin-top: 18px;
+}
+
+// Pod stats
+.segment-selection {
+  select,
+  [type=submit] {
+    margin-top: 10px;
```

Comment:
Do we need this css rule actually? When I turn it off it feels better, aligned with "Displaying results from the Daily segment" line. For example, for `/admins/weekly_user_stats` we align the selection box with the heading. So why having it lower here?